### PR TITLE
[Feature] Add Omnisource training configs

### PR DESCRIPTION
**Pull request by kennymckormick**

### CODE REVIEW

File: configs/recognition/omnisource/README.md [hunk 1]
```diff
@@ -17,7 +17,100 @@ We currently released 4 models trained with OmniSource framework, including both
...
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+  </tr>
+</tbody>
+</table>
```
Comment (dreamerlin) [inline configs/recognition/omnisource/README.md:106]: Sth wrong with the CSS, https://github.com/open-mmlab/mmaction2/blob/7d5bba27609c8d86d90d7765de6d9b9a303c7fa1/configs/recognition/omnisource/README.md

File: configs/recognition/omnisource/slowonly_r50_8x8x1_256e_minikinetics/slowonly_r50_8x8x1_256e_minikinetics_kineticsraw_rgb.py [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+        norm_eval=False),
+    cls_head=dict(
+        type='I3DHead',
+        in_channels=2048,
+        num_classes=200,
+        spatial_type='avg',
+        dropout_ratio=0.5))
+# model training and testing settings
+train_cfg = None
+test_cfg = dict(average_clips=None)
```
Comment (dreamerlin) [inline configs/recognition/omnisource/slowonly_r50_8x8x1_256e_minikinetics/slowonly_r50_8x8x1_256e_minikinetics_kineticsraw_rgb.py:22]: 'prob' for 3D models

File: configs/recognition/omnisource/slowonly_r50_8x8x1_256e_minikinetics/slowonly_r50_8x8x1_256e_minikinetics_kineticsraw_rgb.py [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    type='SGD', lr=0.15, momentum=0.9,
+    weight_decay=0.0001)  # this lr is used for 8 gpus
+optimizer_config = dict(grad_clip=dict(max_norm=40, norm_type=2))
+# learning policy
+lr_config = dict(policy='CosineAnnealing', min_lr=0)
+total_epochs = 256
+checkpoint_config = dict(interval=8)
+workflow = [('train', 8)]
+evaluation = dict(
+    interval=8, metrics=['top_k_accuracy', 'mean_class_accuracy'], topk=(1, 5))
```
Comment (dreamerlin) [inline configs/recognition/omnisource/slowonly_r50_8x8x1_256e_minikinetics/slowonly_r50_8x8x1_256e_minikinetics_kineticsraw_rgb.py:139]: ~, topk=(1, 5)~

File: configs/recognition/omnisource/slowonly_r50_8x8x1_256e_minikinetics/slowonly_r50_8x8x1_256e_minikinetics_webimage_rgb.py [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+# optimizer
+optimizer = dict(
+    type='SGD', lr=0.15, momentum=0.9,
+    weight_decay=0.0001)  # this lr is used for 8 gpus
+optimizer_config = dict(grad_clip=dict(max_norm=40, norm_type=2))
+# learning policy
+lr_config = dict(policy='CosineAnnealing', min_lr=0)
+total_epochs = 256
+checkpoint_config = dict(interval=8)
+workflow = [('train', 8)]
```
Comment (dreamerlin) [inline configs/recognition/omnisource/slowonly_r50_8x8x1_256e_minikinetics/slowonly_r50_8x8x1_256e_minikinetics_webimage_rgb.py:137]: Curious about `workflow = [('train', 8)]`

File: configs/recognition/omnisource/README.md [hunk 5]
```diff
@@ -17,7 +17,100 @@ We currently released 4 models trained with OmniSource framework, including both
...
 
-We will soon release the web dataset and training code used by OmniSource.
+### Benchmark on Mini-Kinetics
+
+We release a subset of web dataset used in the OmniSource paper. Specifically, we release the web data in the 200 classes of [Mini-Kinetics](https://arxiv.org/pdf/1712.04851.pdf). The statistics of those datasets is detailed in [preparing_omnisource](/tools/data/omnisource/preparing_omnisource.md). To obtain those data, you need to fill in a [data request form](https://docs.google.com/forms/d/e/1FAIpQLSd8_GlmHzG8FcDbW-OEu__G7qLgOSYZpH-i5vYVJcu7wcb_TQ/viewform?usp=sf_link). After we received your request, the download link of these data will be send to you. For more details on the released OmniSource web dataset, please refer to [preparing_omnisource](/tools/data/omnisource/preparing_omnisource.md).
```
Comment (innerlee) [inline configs/recognition/omnisource/README.md:22]: Should we add Mini-Kinetics in the `preparing datasets` documentation?

File: configs/recognition/omnisource/README.md (hunk 1 above)
Comment (kennymckormick) [reply configs/recognition/omnisource/README.md:106]: OK, I will remove this part.

File: configs/recognition/omnisource/README.md (hunk 5 above)
Comment (kennymckormick) [reply configs/recognition/omnisource/README.md:22]: No need to do so, it is included in the OmniSource data releasing.

File: configs/recognition/omnisource/slowonly_r50_8x8x1_256e_minikinetics/slowonly_r50_8x8x1_256e_minikinetics_webimage_rgb.py (hunk 4 above)
Comment (kennymckormick) [reply configs/recognition/omnisource/slowonly_r50_8x8x1_256e_minikinetics/slowonly_r50_8x8x1_256e_minikinetics_webimage_rgb.py:137]: Changed to ('train', 1). Actually, it makes no difference as long as can be divided by total_epochs

File: configs/recognition/omnisource/README.md [hunk 6]
```diff
@@ -17,7 +17,90 @@ We currently released 4 models trained with OmniSource framework, including both
...
+
+We benchmark the OmniSource framework on the released subset, results are listed in the following table (we report the Top-1 and Top-5 accuracy on Mini-Kinetics validation). The cbenchmark can be used as a baseline for video recognition with web data.
+
+<table class="tg">
```
Comment (innerlee) [inline configs/recognition/omnisource/README.md:26]: > `class="tg"`

i guess these classes will be ignored by github. you can remove then

Comment (innerlee) [reply configs/recognition/omnisource/README.md]: actually, i meant all css classes :)

**Comment by innerlee** [comment]
changelogs are very easy to get conflicted. one way is to insert your line random in the section, rather than appending to the end.
this will help git diff tools

**Comment by innerlee** [comment]
btw, remove all the css classes in the table

**Comment by kennymckormick** [comment]
@innerlee  resolved

**Review by innerlee — APPROVED** [review]

**Comment by innerlee** [comment]
hmm

**Pull request merged**